playbooks/common/openshift-cluster/openshift_metrics.yml

```diff
@@ -1,4 +1,11 @@
 ---
+# I believe this violates our rule of either includes or plays.
+# Unfortunately we've documented calling this playbook directly so
+# I'd really like to apply the group mappings here rather than up in byo
+- include: ../../common/openshift-cluster/std_include.yml
```

Comment:
are we doing something like this in `playbooks/common/openshift-cluster/openshift_logging.yml` ? Why wouldn't we have this in the `byo` metrics playbook?